### DISCUSSION TITLE
Brightness indicator not showing with app NotchNook

### DISCUSSION THREAD

**Discussion by DenizUe** [MonitorControl/MonitorControl#1676] [Q&A] [answered]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

I got a new App called [NotchNook](https://lo.cafe/notchnook) which also changes the volume and brightness indicators/animations. But when I'm connected to my external monitor in clamshell mode the brightness animation is not working and the there is no indication at all. The brightness does change, I just don't get the native indicator or the new animation from NotchNook. 
Also [reported](https://feedback.notchnook.cafe/de/p/brightness-animation-not-working-on-external-monitor) this bug to their team

### Steps to reproduce

1. Run MonitorControl
2. Run NotchNook
3. Change brightness

### Expected behavior

Expected to see the animation from NotchNook

### Environment Information (please complete the following information)

```markdown
- macOS version: 15.2 Beta 2
- Mac model:Air 2022
- MonitorControl version: 4.2.0 Build 7048
- Monitor(s): 1x Samsung
- Apple Silicon/M1 (yes or no): yes M2
```

**Comment by waydabber** [comment, marked as answer]
I don't plan adding NotchNook support to MonitorControl. My other app BetterDisplay supports MediaMate and I am happy to add NotchNook support as well. Actually the dev of NotchNook can integrate BetterDisplay easily as it has an API just for the purpose:

https://github.com/waydabber/BetterDisplay/wiki/Integration-features,-CLI#osd-notification-dispatch-integration

If you are in contact with the NotchNook dev, let him know I am happy to cooperate.
